packages/frontend/src/features/tableCalculation/components/SqlForm.tsx

```diff
@@ -42,22 +46,26 @@ export const SqlForm: FC<Props> = ({ form, isFullScreen }) => {
     const { setAceEditor } = useExplorerAceEditorCompleter();
     return (
         <>
-            <SqlEditor
-                mode="sql"
-                theme="github"
-                width="100%"
-                placeholder={SQL_PLACEHOLDER}
-                maxLines={isFullScreen ? 40 : 20}
-                minLines={isFullScreen ? 40 : 8}
-                editorProps={{ $blockScrolling: true }}
-                onLoad={setAceEditor}
-                enableLiveAutocompletion
-                enableBasicAutocompletion
-                wrapEnabled
-                isFullScreen={isFullScreen}
-                gutterBackgroundColor={theme.colors.gray['1']}
-                {...form.getInputProps('sql')}
-            />
+            <ScrollArea h={isFullScreen ? '95%' : '150px'}>
```

Comment:
I was a bit concerned about 95% and where the rest of the 5% go - but it works well even when resizing windows

Reply:
Yea, same here! Not a fan of these percentages but they do work well sometimes 😅